docs/resources/aviatrix_transit_gateway.md

```diff
@@ -320,60 +323,81 @@ resource "aviatrix_transit_gateway" "test-edge-transit-1" {
         index  = 0
     }
     interfaces {
-        gateway_ip   = "192.168.22.1"
-        ip_address   = "192.168.22.11/24"
-        type         = "WAN"
-        index        = 2
+        gateway_ip  = "192.168.22.1"
+        ip_address  = "192.168.22.11/24"
+        type        = "WAN"
+        index       = 2
     }
     interfaces {
-        gateway_ip   = "192.168.23.1"
-        ip_address   = "192.168.23.11/24"
-        type         = "WAN"
-        index        = 3
+        gateway_ip = "192.168.23.1"
+        ip_address = "192.168.23.11/24"
+        type       = "WAN"
+        index      = 3
     }
-}
-```
-```hcl
-#Create an Aviatrix Edge Transit HA Gateway in AEP
-resource "aviatrix_transit_gateway" "test-edge-transit-1-hagw" {
-    cloud_type = 262144
-    account_name = "edge_aep"
-    gw_name = "test-edge-transit-1"
-    vpc_id = "site-1"
-    gw_size = "SMALL"
-    device_id = "0ec1266c-3877-40b7-8c61-8245d69e394a"
-    site_id = "site-3"
-    peer_backup_port = "eth1"
-    connection_type = "private"
-    interfaces {
-        gateway_ip  = "192.168.20.1"
-        name        = "eth0"
-        ip_address  = "192.168.20.12/24"
-        type        = "WAN"
+    interface_mapping{
+      name  = "eth0"
+      type  = "wan"
+      index = 0
     }
-    interfaces {
-        gateway_ip                  = "192.168.21.1"
-        name                        = "eth1"
-        ip_address                  = "192.168.21.12/24"
-        type                        = "WAN"
+    interface_mapping{
+      name  = "eth1"
+      type  = "wan"
```

Comment:
The type of the interface is different for interface config and interface mapping.
I know it is the same like this in the API as well.
But do you want to make it common at both places in TF in upper case like:
"WAN". "LAN", "MANAGEMENT"

Reply:
Updated the interface mapping to match interface config type name